src/components/DigitalCredentials/IssueCredentials.vue

```diff
@@ -0,0 +1,130 @@
+<template>
+  <section id="issue-credentials" class="py-8">
```

Comment:
For maintaining layouts, it's better if the top element (here, section) doesn't have layout CSS -- move it to the component that uses this instead.

Reply:
Good call, removed all step components and applied to router-view